app/util/parseQRCode.js

```diff
@@ -27,10 +35,13 @@ const parseQRCode = data => {
     if (!asset) throw UNRECOGNIZED_ASSET
   }
 
+  const parsedAmount = searchParams.get('amount')
+  const amount = parsedAmount ? Number(parsedAmount) : null
```

Comment:
This could be fine, but might want to use big number here. Not sure exactly the use case and if it is needed. Just something to think about

Reply:
You're totally right 😓Returned to original `?string` to avoid any regression risk fab0388